src/measure_theory/measure/measure_space.lean

```diff
@@ -2455,6 +2444,34 @@ lemma sigma_finite.of_map (μ : measure α) {f : α → β} (hf : measurable f)
    λ n, by simp only [← map_apply hf, measurable_spanning_sets, measure_spanning_sets_lt_top],
    by rw [← preimage_Union, Union_spanning_sets, preimage_univ]⟩⟩⟩
 
+lemma _root_.measurable_equiv.sigma_finite_map {μ : measure α} (f : α ≃ᵐ β) (h : sigma_finite μ) :
+  sigma_finite (map f μ) :=
+by { refine sigma_finite.of_map _ f.symm.measurable _,
+     rwa [map_map f.symm.measurable f.measurable, f.symm_comp_self, measure.map_id] }
+
+/-- Similar to `ae_of_forall_measure_lt_top_ae_restrict`, but where you additionally get the
+  hypothesis that another σ-finite measure has finite values on `s`. -/
+lemma ae_of_forall_measure_lt_top_ae_restrict' {μ : measure α} (ν : measure α) [sigma_finite μ]
+  [sigma_finite ν] (P : α → Prop)
+  (h : ∀ s, measurable_set s → μ s < ∞ → ν s < ∞ → ∀ᵐ x ∂(μ.restrict s), P x) :
+  ∀ᵐ x ∂μ, P x :=
```

Comment:
Actually, you can add countably many assumptions of this type. I don't know whether we'll ever need this but IMHO it's worth mentioning in a comment.

Reply:
Why is that true?

Reply:
Probably, I was wrong. You can only add finitely many assumptions (the same as adding one assumption about the sum of these measures).